opensrp-unicef-tunisia/src/main/java/org/smartregister/uniceftunisia/interactor/LoginInteractor.java

```diff
@@ -60,8 +61,10 @@ protected void scheduleJobsPeriodically() {
         RecurringIndicatorGeneratingJob.scheduleJob(RecurringIndicatorGeneratingJob.TAG,
                 TimeUnit.HOURS.toMinutes(6), getFlexValue(BuildConfig.DATA_SYNC_DURATION_MINUTES));
 
-        // Schedule vaccine schedules update after midnight
-        AppVaccineUpdateJob.scheduleEverydayAt(AppVaccineUpdateJob.TAG, 1, 7);
+        ArchiveClientsJob.scheduleDaily();
+
+        // Schedule daily jobs
+        AppVaccineUpdateJob.scheduleEverydayAt(AppVaccineUpdateJob.TAG, 1, 20);
```

Comment:
do you need two call for this?

Reply:
The archive job runs once after login then scheduled daily starting from the next day.